oozie/src/main/java/org/apache/falcon/workflow/engine/OozieWorkflowEngine.java

```diff
@@ -153,6 +153,10 @@ public void schedule(Entity entity, Boolean skipDryRun, Map<String, String> supp
         if (StartupProperties.isServerInSafeMode()) {
             throwSafemodeException("SCHEDULE");
         }
+        //Adding group information to pass to oozie
```

Comment:
This ties it up to Unix groups. But, I guess it ok for now

Reply:
ok